README.md

```diff
@@ -63,7 +63,6 @@ As of April 5, 2017, Parse, LLC has transferred this code to the parse-community
 
  [parse.com]: https://parseplatform.github.io/
  [guide]: http://parseplatform.github.io/docs/android/guide/
- [blog]: https://blog.parse.com/
```

Comment:
Can you update also parseplatform.github.io to parseplatform.org and parseplatform.github.io/docs/android/guide/ to docs.parseplatform.oog/android/guide?